index.html

```diff
@@ -30,10 +30,22 @@
     window.tido = new window.Tido({
       globalTree: true,
       panels: [
-        {
+        { 
           entrypoint: {
             url: 'https://api.ahiqar.sub.uni-goettingen.de/textapi/ahiqar/syriac/collection.json',
             type: 'collection'
+          },
+          colors: {
+            primary: '#1E40AF'
```

Comment:
I don't think we should allow to set the color per panel. This should be a global setting.